airflow/api_connexion/openapi/v1.yaml

```diff
@@ -604,6 +604,40 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /dags/{dag_id}/dagRuns/updateDagRunState:
```

Comment:
This endpoint feels wrong to me. Since the operation only affects one single DagRun, I feel it should be something like `/dags/{dag_id}/dagRuns/{dag_run_id}/state`. And it should only take a single `state` field (not `dag_id` or `dag_run_id`).